mirgecom/simutil.py

```diff
@@ -92,6 +92,33 @@ def get_number_of_tetrahedron_nodes(dim, order, include_faces=False):
         nnodes = nnodes + (dim+1)*get_number_of_tetrahedron_nodes(dim-1, order)
     return nnodes
 
+def get_box_mesh(dim,a,b,n, t=None, periodic=None):
+    """
+    Creates a rectangular "box" like mesh using function generate_regular_rect_mesh
+
+    Ensures parameters needed for generate_regular_rect_mesh are satisfied
+    a,b,n,periodic are tuples of dimension = dim and are converted if not in this format
```

Comment:
If you could format the arguments like [this](https://github.com/illinois-ceesd/mirgecom/blob/27130a33b32f6a08f1af02dcbb5c34f1410b8b9b/mirgecom/simutil.py#L160-L185), then they'll render better in the built documentation ([like this](https://mirgecom.readthedocs.io/en/latest/support/tools.html)).